src/main/java/twitter4j.properties

```diff
@@ -1,5 +1,5 @@
 debug = true
-oauth.accessToken = 1009436937760006144-lMYADUmKvQShEMFByAQTsu11LfOKNP
-oauth.consumerSecret = R9UYKzyz7liKBdliAROoqw0uqI8d0H9IoMyV8R4OYRZIxzV8Dw
-oauth.consumerKey = gTFj02yhApZALpkmGukOc8UnN
-oauth.accessTokenSecret = RN51oxqnOAPH2mzS7LhkaKtfdxlwLe8ZlX65xihJpfci5
+oauth.accessToken =
```

Comment:
you may want to consider deleting the old keys and generating new ones so your keys don't make it out publicly